src/rng.rs

```diff
@@ -92,9 +84,9 @@ impl<D: Digest> FiatShamirRng for FiatShamirChaChaRng<D> {
     #[inline]
     fn from_seed<'a, T: 'a + ToBytes>(seed: &'a T) -> Self {
         let mut bytes = Vec::new();
-        seed.write(&mut bytes).expect("failed to convert to bytes");
+        seed.write(&mut bytes).unwrap_or(());
```

Comment:
In reality none of the assertions can fail. "seed" is always an implementer of ToBytes and the output of D::digest(&bytes) is always 32 bytes. Maybe we can simply unwrap() in line 87 and 89 and put a "static assertion" that D::OutputSize == 32